tensorrt_llm/_torch/pyexecutor/py_executor.py

```
@@ -3176,19 +3262,38 @@ def _check_disagg_gen_transfer_status(self):
+        # the deferral path in _handle_errors relies on a future
+        # DISAGG_TRANS_ERROR transition to surface the failure: a request
+        # that never times out on the C++ side never moves to error and
+        # would wedge here forever. See _DISAGG_KV_TRANSFER_FALLBACK_DEADLINE_MS.
+        if configured_timeout_ms is None:
+            timeout_ms = _DISAGG_KV_TRANSFER_FALLBACK_DEADLINE_MS
+            using_fallback = True
+        else:
+            timeout_ms = configured_timeout_ms
+            using_fallback = False
 
         def flag_if_kv_transfer_timed_out(req: LlmRequest, type: str) -> None:
```

Ruff (0.15.12)

[error] Function argument `type` is shadowing a Python builtin

(A002)